alphadia/constants/default.yaml

```diff
@@ -96,7 +96,7 @@ custom_modifications:
     composition: C(12)
 
 search:
-  channel_filter: '0'
+  channel_filter: "" # will be mapped to "0" -> load all channels
```

Comment:
That's not 100% right. `channel_filter: ""` will only load the default unmultiplexed channel which is the intended behaviour. So we just need to change the comment.